src/components/biggive-footer/biggive-footer.scss

```diff
@@ -11,7 +11,7 @@ nav {
   width: calc(33.3% - 20px);
   margin: 0;
   padding: 0;
-  h5 {
+  .heading {
     @include heading-5();
```

Comment:
If we use a `headingLevel` that isn't 5, the component will still always `@include heading-5()`. Are you okay with this?

Reply:
Yes, see https://github.com/thebiggive/components/pull/111#discussion_r1064630260 . (sorry I was reading backwards)